showcase.html

```diff
@@ -0,0 +1,295 @@
+{% extends "base.html" %}
```

Comment:
This commit looks incorrect - the templates should be located in `views/` directory.

Reply:
I think that I already removed this file. I made mistake while uploading this file.